cloth-diapers/cloth-diapers-basics.md

```diff
@@ -0,0 +1,296 @@
+## Cloth Diapers
+
+There are many sections that need to be written up in this document, but I'll
+start by writing up the critical sections required for basic knowledge
+or those that I have a specific opinion about.
+
+_Disclaimer_: These are my personal opinions and as such, don't come at me if
+yours are different!
+
+However, the plan is to include the following
++ Nomenclature (multiple languages)
++ [Types of Cloth Diapers](#types-of-cloth-diapers)
++ [Materials and Absorbency](#materials-and-absorbency)
++ [Newborn vs One Size](#newborn-vs-one-size)
++ Velcro vs Poppers
++ [Our Choices](#our-choices)
++ Things to Know
++ What causes leaks
+
+## Types of Cloth Diapers
+![image](https://user-images.githubusercontent.com/4185328/169092681-4b44aaef-1127-4937-b906-1b2ddc864821.png)
+
+I don't want to go into all the different types of cloth diaper just yet as
+there are actually a lot of decent resources online already on this, such as
+https://www.thenappygurus.com/types-of-cloth-nappies.html
+
+The whole of the [Nappy Gurus Help & Advice](https://www.thenappygurus.com/help.html)
+section is extremely well put together and very helfpul if you're starting out.
+
+I'm just leaving this diagram here as a reference point to discuss the
+choices we have made (see below).
+
+## Materials and Absorbency
+
+<img width="664" alt="image" src="https://user-images.githubusercontent.com/4185328/169095781-8c3c8f69-3ae5-4e04-ab39-780f198651a5.png">
+
+Cloth diapers hinge on both their speed of and capacity for absorbency.
+
+The only other thing to mention other than the table above is 'Charcoal Bamboo'.
+This is actually not bamboo at all, it's just a fancy name for microfibre and
+equally unhelpful.
+
+### Absorbency capacity
+The more capacity for absorbency, the more liquid a diaper will hold.
+
+Having said this, every baby is different so whilst some can get away with
```

Comment:
This sentence could be _significantly_ condensed; It's 55 words / 344 characters. i.e. more than 2 tweets! 😮 

![image](https://user-images.githubusercontent.com/194400/173175553-cc97e505-e14d-4795-bacf-018d25cf72a2.png)

Ref: https://insidegovuk.blog.gov.uk/2014/08/04/sentence-length-why-25-words-is-our-limit/

Every baby may _appear_ different to their parents ... 
but they all have similar traits and belong to a distribution curve. 

You could break up the mega sentence into chunks:
Each baby will urinate a varying amount depending on their level of hydration/nursing.